### DISCUSSION TITLE
Support for configuration dotfile?

### DISCUSSION THREAD

**Discussion by dannysteenman** [biomejs/biome#1889] [Ideas]
Biome currently support `biome.json` and `biome.jsonc` as configuration files. It would be nice to have additional support for dotfiles e.g. `.biome.json` similar to how eslint supports multiple configuration formats: https://eslint.org/docs/latest/use/configure/configuration-files#configuration-file-formats

**Comment by ematipico** [comment]
Eslint is moving away from having all these formats: https://eslint.org/blog/2023/11/whats-coming-in-eslint-9.0.0/#flat-config-now-the-default-and-has-some-changes

The reason why we added `biome.jsonc` is for people to add comments in their configuration file. I don't see any particular strong reason to add a dot file.
